appserver/admingui/common/src/main/resources/js/adminjsf.js

```diff
@@ -2399,8 +2399,9 @@ admingui.ajax = {
         }
 
         // FIXME: These 2 functions (should) only need be replaced after FPR...
-        webui.suntheme.hyperlink.submit = admingui.woodstock.hyperLinkSubmit;
-        webui.suntheme.jumpDropDown.changed = admingui.woodstock.dropDownChanged;
+        // FIXME: The next 2 lines were commented because webui is not recognized and the code breaks with it
+        // webui.suntheme.hyperlink.submit = admingui.woodstock.hyperLinkSubmit;
+        // webui.suntheme.jumpDropDown.changed = admingui.woodstock.dropDownChanged;
```

Comment:
Have you tried doing what Glassfish [did](https://github.com/eclipse-ee4j/glassfish/blob/c822239824cf4b2dc1f9aaac1790b2f6c6c25595/appserver/admingui/common/src/main/resources/js/adminjsf.js#L2411)?